CMakeLists.txt

```diff
@@ -159,10 +160,15 @@ endif()
 if( TRACCC_BUILD_SYCL )
    add_subdirectory( device/sycl )
 endif()
-add_subdirectory( examples )
 add_subdirectory( io )
 add_subdirectory( plugins )
 
+if ( TRACCC_BUILD_EXAMPLES )
+   # Set up Boost.
+   find_package( Boost 1.75.0 REQUIRED COMPONENTS program_options)
```

Comment:
Why did you impose this 1.75 version minimum? I don't see anything in your code that would not have been part of this part of Boost ever since I've been using it.

Reply:
I dumbly put the version number that I was using. What is the best way figuring out this?

Reply:
You don't need to specify anything. You can just write:

```cmake
find_package( Boost REQUIRED COMPONENTS program_options )
```

If you're really curious, you can check what version was actually found using the `${Boost_VERSION}` variable set up by this call. But I doubt that our code needs to know it.

The version number specified for `find_package(...)` is treated as a minimum version that the code needs. Or an exact one if you use the `EXACT` keyword, like:

```cmake
find_package( Boost 1.75.0 EXACT REQUIRED )
```

Long story short, just remove the version number. :wink: